app/lib/taric_importer.rb

```diff
@@ -26,6 +26,7 @@ def import
     return unless proceed_with_import?(filename)
```

Comment:
These changes may not be required as Taric Updates and importing is processed in app/lib/tariff_synchronizer/taric_update_downloader.rb.  It could be that this code is actually redundant now...